tests/Tools/HelperTrait/ProductTrait.php

```diff
@@ -313,6 +313,120 @@ public function get_sample_attributes(): array {
 		];
 	}
 
+	/**
+	 * @return array
+	 */
+	public function get_sample_rules(): array {
+		return [
+			[
+				'attribute'               => 'gtin',
+				'source'                  => '012345678905',
+				'category_condition_type' => 'ALL',
+				'categories'              => '',
+			],
+			[
+				'attribute'               => 'mpn',
+				'source'                  => 'GO54321OOGLE',
+				'category_condition_type' => 'ALL',
+				'categories'              => '',
+			],
+			[
+				'attribute'               => 'brand',
+				'source'                  => 'acme',
+				'category_condition_type' => 'ALL',
+				'categories'              => '',
+			],
+			[
+				'attribute'               => 'condition',
+				'source'                  => 'new',
+				'category_condition_type' => 'ALL',
+				'categories'              => '',
+			],
+			[
+				'attribute'               => 'gender',
+				'source'                  => 'unisex',
+				'category_condition_type' => 'ALL',
+				'categories'              => '',
+			],
+			[
+				'attribute'               => 'size',
+				'source'                  => 'M',
+				'category_condition_type' => 'ALL',
+				'categories'              => '',
+			],
+			[
+				'attribute'               => 'sizeSystem',
+				'source'                  => 'US',
+				'category_condition_type' => 'ALL',
+				'categories'              => '',
+			],
+			[
+				'attribute'               => 'sizeType',
+				'source'                  => 'regular',
+				'category_condition_type' => 'ALL',
+				'categories'              => '',
+			],
+			[
+				'attribute'               => 'color',
+				'source'                  => 'blue',
+				'category_condition_type' => 'ALL',
+				'categories'              => '',
+			],
+			[
+				'attribute'               => 'pattern',
+				'source'                  => 'squares',
+				'category_condition_type' => 'ALL',
+				'categories'              => '',
+			],
+			[
+				'attribute'               => 'ageGroup',
+				'source'                  => 'newborn',
+				'category_condition_type' => 'ALL',
+				'categories'              => '',
+			],
+			[
+				'attribute'               => 'multipack',
+				'source'                  => '0',
```

Comment:
Isn't it better to test with an actual multipack value here? `'0'` is a falsey value, and assertEquals doesn't strictly type check so I'm not sure how reliable that check would be (possibly change to assertSame)?

Maybe some specific tests to confirm `format_attribute` is mapping all the value.